pkg/core/engine/main.go

```diff
@@ -30,13 +28,12 @@ type Engine struct {
 
 // Clean remove every traces from an updatecli run
 func (e *Engine) Clean() (err error) {
-	tmp.Clean()
-	return err
+	err = tmp.Clean()
+	return
```

Comment:
That's interesting that you use the implicit return, without specifying `return err`